<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spell_Edit</class>
 <widget class="QDialog" name="Spell_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fID"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Level</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fLevel"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fClass">
     <item>
      <property name="text">
       <string>No class</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fighter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cleric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Magic user</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thief</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Monster</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fTarget">
     <item>
      <property name="text">
       <string>Single neighbour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area neighbour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Self</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single range projectile</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Caster area</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fRange"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fRadius"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Spell_Edit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Spell_Edit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
